vcl_templates/www.vcl.erb

```diff
@@ -220,6 +220,10 @@ sub vcl_recv {
     set req.http.GOVUK-ABTest-EducationNavigation = "A";
   } else if (req.url ~ "[\?\&]ABTest-EducationNavigation=B(&|$)") {
     set req.http.GOVUK-ABTest-EducationNavigation = "B";
+  } else if (req.url ~ "^\/education(\/)?.*$") {
```

Comment:
I am not sure, but it does sound like a possibility. What do you think of rolling back to the more verbose regex: `"^\/education(\/)?.*$"`? That way we guarantee that we set the cookie only for:
- `/education`
- `/education/`
- `/education/<any-taxon-base-path>`

And we know there are no content pages nested under `/education`.